accounts/migrations/0044_new-analytics-permissions.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.23 on 2019-10-23 16:02
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("accounts", "0043_gestational-age-can-be-blank")]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name="child",
+            options={
+                "ordering": ["-birthday"],
+                "permissions": [
+                    ("can_view_child_analytics_data", "Can View Child Analytics Data")
+                ],
+            },
+        ),
+        migrations.AlterModelOptions(
+            name="demographicdata",
+            options={
+                "ordering": ["-created_at"],
+                "permissions": [
+                    (
+                        "can_view_demographics_analytics_data",
```

Comment:
we don't use this, correct? (see comment below about why this might be complicated anyway)

Reply:
See above